rename_imports.py

```diff
@@ -0,0 +1,78 @@
+
+from LSP.plugin.core.file_watcher import FileWatcherEvent, FileWatcherProtocol, get_file_watcher_implementation, FileWatcher
+from LSP.plugin.core.typing import  List
+import os
+import sublime
+
+
+def register_rename_import_file_watcher(root_path: str):
+    file_watcher = get_file_watcher_implementation()
+    if file_watcher:
+        watcher = file_watcher.create(
+            root_path=root_path,
+            patterns=["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx"],
+            events=["create", "delete"],
+            ignores=['**/node_modules/**'],
+            handler=HandleRenameImport
```

Comment:
You need to pass an initialized instance.

Reply:
I tired that, but got a `LSP-file-watcher-chokidar: ERROR: on_payload(): Handler already deleted`

Reply:
based on the code, I concluded that I should not pass an instance

LSP-file-watcher-chokidar will create the instance:
https://github.com/sublimelsp/LSP-file-watcher-chokidar/blob/main/watcher.py#L230

Reply:
It doesn't create an instance. It's a weak reference to an instance so you need to call it to get the reference.

It's being deleted immediately, that's why you are getting "Handler already deleted". You need to store a reference to it somewhere, otherwise it will be GC-ed. That's also related to my other comment about having to bound the controller to the running server instance.